admin_guide/manage_authorization_policy.adoc

```diff
@@ -565,3 +565,96 @@ administrator, you can grant them the abilities.
 ----
 $ oadm policy add-role-to-user system:daemonset-admin <user>
 ----
+
+[[manage-authorization-policy-creating-local-role]]
+== Creating a Local Role
+
+To create a local role for a project, you must build it from scratch to
```

Comment:
Perhaps "We recommend you build it from scratch rather than copying an existing role".

@enj?